recipes/pygsl_lite/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pygsl_lite" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygsl_lite-{{ version }}.tar.gz
+  sha256: 5f72ba936e49eb6dafd5fb78668e50538b8ef9d1e321d937018c5cad0b4227a4
+
+build:
+  skip: true  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - setuptools-scm >=3.4.3
+    - wheel
+    - numpy >=1.23
+    - pip
+    - gsl >=2.7
```

Comment:
You should remove this version pin so that conda-forge can specify which version to use. At the moment the pinned version is 2.7 anyway:

```suggestion
    - gsl
```

Reply:
thanks, removed